webui/react/src/pages/DesignKit.tsx

```diff
@@ -202,38 +218,190 @@ const DropdownsSection: React.FC = () => {
+            Select dropdowns render in their own layer by default to ensure they are not clipped by
+            containers with overflow: hidden or overflow: scroll. This causes extra difficulty for
+            people who use screen readers, so we recommend rendering the ComboBox options dropdown
+            inline unless they are in overflow containers.
           </li>
         </ul>
         <strong>Truncation</strong>
         <ul>
           <li>
-            By default, the ComboBox truncates option text instead of wrapping to a new line.
-            Because this can lose meaningful information, it is recommended to adjust styles to wrap
-            the option text.
+            By default, the Select truncates option text instead of wrapping to a new line. Because
+            this can lose meaningful information, it is recommended to adjust styles to wrap the
+            option text.
           </li>
         </ul>
       </AntDCard>
       <AntDCard title="Usage">
-        <strong>Default dropdown</strong>
-        <SelectFilter
-          defaultValue={1}
+        <strong>Default Select</strong>
+        <Select
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Variations</strong>
+        <strong>Select with default value</strong>
+        <Select
+          defaultValue={2}
           options={[
             { label: 'Option 1', value: 1 },
             { label: 'Option 2', value: 2 },
             { label: 'Option 3', value: 3 },
           ]}
         />
-        <strong>Disabled dropdown</strong>
-        <SelectFilter
+        <strong>Select with placement</strong>
+        {placementOptions.map((placement) => (
+          <Select
+            key={placement[1]}
+            label={placement[1]}
+            options={[
+              { label: 'Option 1', value: 1 },
+              { label: 'Option 2', value: 2 },
+              { label: 'Option 3', value: 3 },
+            ]}
+            placement={placement[0]}
+          />
+        ))}
+        <strong>Select with label</strong>
+        <Select
+          label="Select Label"
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+        />
+        <strong>Select with placeholder</strong>
+        <Select
+          options={[
+            { label: 'Option 1', value: 1 },
+            { label: 'Option 2', value: 2 },
+            { label: 'Option 3', value: 3 },
+          ]}
+          placeholder="Select Placeholder"
+        />
+        <strong>Disabled Select</strong>
+        <Select
           defaultValue="disabled"
           disabled
           options={[{ label: 'Disabled', value: 'disabled' }]}
         />
-        <hr />
+        <strong>Select with search</strong>
```

Comment:
imo this should be `Select without search` because search is enabled by default. This title sounds like i can enable search, but up to you

Reply:
this is a good call I have made the update!